prboom2/src/sounds.c

```diff
@@ -247,4 +247,210 @@ sfxinfo_t S_sfx[] = {
   //e6y
   { "secret", false, 60, 0, -1, -1, 0 },
   { "gibdth", false, 60, 0, -1, -1, 0 },
+  // Everything from here up to 400 is reserved for future use.
+
+  // Free slots for DEHEXTRA. Priorities should be overridden by user.
+  // The gap to accomodate Eternity means I must use desginated initializers,
+  // or otherwise supply an exact number of dummy entries to pad it out.
+  // Not sure which would be uglier to maintain. -SH
+  [500] = { "fre000", false, 127, 0, -1, -1, 0 },
```

Comment:
Hm, is the "freXXX" name scheme agreed upon with other ports?

Reply:
I posted the naming scheme in the DEHEXTRA thread on Doomworld hoping to receive comment, but only really heard back from Altazimuth of Eternity Engine, who I believe has already implemented the sounds with these lump names. Who else should I pester?

Reply:
I see. Could you probably add a link to the relevant part of the thread in the comment? Or a citation of Altazimuth's reply? Or a link to the commit which adds this code to the Eternity Engine?  Because without this knowledge, the chosen naming scheme leaves me a bit puzzled.

Reply:
So?

Reply:
Hm, forum links are pretty long these days. I guess it would be best to add something like "This implements prior art added to The Eternity Engine in commit https://github.com/team-eternity/eternity/commit/b8fb8f71." or similar to the comment (I'm not a native speaker, so please bear with me). Then I'd be fine with this PR.

Reply:
Comment adjusted to include the URL of that commit.